src/screens/SettingsScreen.js

```diff
@@ -24,15 +22,7 @@ const { MATOMO_TRACKING } = consts;
 
 const keyExtractor = (item, index) => `index${index}-id${item.id}`;
 
-const renderSectionHeader = ({ section: { title } }) =>
-  !!title && (
-    <View>
-      <TitleContainer>
-        <Title accessibilityLabel={`(${title}) ${consts.a11yLabel.heading}`}>{title}</Title>
-      </TitleContainer>
-      {device.platform === 'ios' && <TitleShadow />}
-    </View>
-  );
+const renderSectionHeader = ({ section: { title } }) => !!title && <SectionHeader title={title} />;
```

Comment:
how is this related?

Reply:
this is directly related. i stumbled across this while looking at this screen for reference for the filter.